src/routers/task.js

```diff
@@ -52,11 +54,11 @@ router.get('/tasks/:id', auth, async(req, res) => {
     try {
         const task = await Task.findOne({ _id, createdBy: req.user._id })
         if (!task) {
-            return res.status(404).send()
+            return res.status(404).json()
         }
-        res.status(200).send(task)
+        res.status(200).json(task)
```

Comment:
status 200 is default. the same mistake is everywhere.

Reply:
Fixed.